rust/src/mark.rs

```diff
@@ -1,6 +1,6 @@
 use automerge as am;
 
-use crate::ScalarValue;
+use crate::{ScalarValue, Value};
```

Comment:
```suggestion
use crate::ScalarValue;
```

With `KeyValue` being a struct over `ScalarValue` instead of `Value`, I don't think you'll need this inclusion.

Reply:
Please, double check this comment: https://github.com/automerge/automerge-swift/pull/186#discussion_r1636963716 - it seems `map_entries` need to work over `Value`